src/main/java/tech/fastj/engine/package-info.java

```diff
@@ -0,0 +1,5 @@
+package tech.fastj.engine;
+
+/**
+ * Classes used internally inside FastJ.
```

Comment:
> internally inside

This is fairly redundant -- internal and inside both mean the same thing in this context.
Consider: "FastJ's internal engine classes".